en/SUMMARY.md

```diff
@@ -29,21 +29,32 @@
   * [Interfacing to ROS](simulation/ros_interface.md)
   * [AirSim Simulation](simulation/airsim.md)
   * [Multi-Vehicle Simulation](simulation/multi-vehicle-simulation.md)
-* [Tutorials](tutorials/tutorials.md)
-  * [Ground Control Station](qgc/README.md)
-  * [Writing an Application](tutorials/tutorial_hello_sky.md)
-  * [Video Streaming in QGC](qgc/video_streaming.md)
-  * [Long-distance Video Streaming](qgc/video_streaming_wifi_broadcast.md)
+* [Hardware](hardware/README.md)
+  * [Autopilot Hardware](flight_controller/README.md)
+  * [Airframes](airframes/README.md)
+    * [Airframes Reference](airframes/airframe_reference.md)
+    * [Adding a New Airframe](airframes/adding_a_new_frame.md)
+    * [Parrot Bebop](advanced/parrot_bebop.md)
```

Comment:
This is interim. I plan to remove [this doc](https://hamishwillee.gitbooks.io/havdevguide/content/v/1c01c94cbbd1bfcfabe658092ebdda87e1574ed1/en/advanced/parrot_bebop.html) and integrate content into [user guide](https://docs.px4.io/en/flight_controller/bebop.html)